common/faceting.js

```diff
@@ -28,7 +28,11 @@
                 return {
                     selected: (typeof selected == 'boolean') ? selected: false,
                     isNotNull: true,
-                    displayname: {"value": defaultDisplayname.notNull, "isHTML": true}
+                    displayname: {"value": defaultDisplayname.notNull, "isHTML": true},
+                    tooltip: {
+                        value: messageMap.tooltip.notNull
```

Comment:
@jrchudy Is there a reason that you have defined the tooltip as an object here? It seems like you can just say `"tooltip": message.tooltip.notNull`.

Reply:
But in the template you're always using the `.value` without checking for the `isHTML`. Also if you want to be following the same pattern, in here you need to add the `"isHTML": true`. But I don't think it's worth doing so (since as I mentioned we're not checking for it in the template).